ImperatorToCK3/Copy_Files_Release.bat

```diff
@@ -0,0 +1,6 @@
+echo on
```

Comment:
You should also create linux version of this.

Reply:
I think this is already done in Copy_Files.sh
`
rm -rf "Release-Linux/blankMod"

cp -r "Data_Files/blankMod" "Release-Linux/"
`